src/psyclone/tests/psyir/nodes/omp_directives_test.py

```diff
@@ -51,8 +51,8 @@
 
 BASE_PATH = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(
     os.path.abspath(__file__)))), "test_files", "dynamo0p3")
-GOCEAN_BASE_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)),"..","..",
-                                "test_files", "gocean1p0")
+GOCEAN_BASE_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)),
```

Comment:
Not your fault but since you've touched this, please could you fix the use of ".." with the OS-independent `os.pardir` method. Rupert used this somewhere recently so you'll find it if you grep...